src/main/java/uk/gov/justice/digital/delius/data/api/PersonalCircumstance.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDateTime;
+
+@Data
+@Builder
+public class PersonalCircumstance {
+    private Long personalCircumstanceId;
+    private Long offenderId;
+    private KeyValue type;
```

Comment:
Should these be called `type` and `subType` or `personalCircumstanceType` etc ?

Reply:
It's a toughie with anonymous tuples like this. My gut feeling would be to call it 'value' for want of a better vague term

Reply:
yeah, I found a precedence with `Nsi` model, which calls it `nsiType` and `nsiSubType` - so renamed to be consistent.